doc/pages/keys.asciidoc

```diff
@@ -552,7 +552,8 @@ to skim through the jump list using:
     jump backward
 
 *<c-s>*::
-    save selections
+    create a jump step, but also save the current selection to have it be
```

Comment:
Instead of the first part I'd rather say `push current selections to the jump list` because "push" is already used in the existing documentation in the paragraph above.